packages/stripe/src/setup-intent.component.ts

```diff
@@ -92,23 +92,34 @@ export class StripeSetupIntentComponent extends RootComponent implements OnInit
             console.log('running addPaymentMethod')
             let clientSecret = ''
             let publishKey = ''
+            let formMessage = ''
             this.newPaymentMethodPending = true
 
             const model = new Model({
                 target: this.paymentMethodApiPath
             })
+            model.data.setupIntent = true
             await model.save()
             console.log('model', model)
             if (
                 Object.prototype.hasOwnProperty.call(model, 'meta') &&
-                Object.prototype.hasOwnProperty.call(model.meta, 'data') &&
-                Object.prototype.hasOwnProperty.call(model.meta.data, 'clientSecret') &&
-                Object.prototype.hasOwnProperty.call(model.meta.data, 'publishKey') &&
-                !_.isEmpty(model.meta.data.clientSecret) &&
-                !_.isEmpty(model.meta.data.publishKey)
+                Object.prototype.hasOwnProperty.call(model.meta, 'data')
```

Comment:
You can also skip some verbosity with a `_.get(model, 'meta.data.foo')` which will just return `undefined` if the object doesn't go that deep.